app/assets/stylesheets/components/_border.scss

```diff
@@ -10,6 +10,10 @@
 .rounded-xl { border-radius: $border-radius-xl; }
 .rounded-xxl { border-radius: $border-radius-xxl; }
 
+.border-blue-medium {
```

Comment:
I was sort of operating under the assumption that we were changing the colors to match the new styleguide?